app/services/weets.js

```diff
@@ -11,3 +11,12 @@ exports.createWeet = async weetContent => {
     throw databaseError(errorMsg.userFindError);
   }
 };
+exports.getWeets = async () => {
+  try {
+    const allWeets = await Weet.findAll();
+    return allWeets;
```

Comment:
If you are not going to use the `allWeets` variable, you can return the value immediately

```suggestion
    return await Weet.findAll();
```